dbt_sugar/core/task/doc.py

```diff
@@ -35,6 +35,8 @@ def __init__(self, flags: FlagParser, dbt_profile: DbtProfile, config: DbtSugarC
         self._dbt_profile = dbt_profile
         self._sugar_config = config
 
+    # FIXME: Ask Virginia why this is here, I don't think it's needed since we already read the
```

Comment:
@virvirlopez Was wondering why this is here since we already read `DbtProfile` in main. Can you get back to me?

Reply:
to connect to the DB, to run the queries.

Reply:
Yeah but it's already available for you in main so I don't think we need to re-create it here do we?